ci-build.yaml

```diff
@@ -0,0 +1,23 @@
+# this is a CI build that tests changes among build templates
+# if succesfull it tags with a new version (based on branch)
```

Comment:
Would be nice to put a documentation link here describng a tag-name strategy

Reply:
Well tag would follow version from **gitversion** task and also gitflow strategy. Update in comment